### ISSUE TITLE
activation_conf_from_stem returns time twice, no confidence

### ISSUE THREAD

**Issue by bmcfee** [marl/medleydb#31]
Quick example:

... or maybe the reverse.  I'm not sure.

```python
>>> t = next(medleydb.load_all_multitracks())
>>> t.activation_conf_from_stem(0)[:10]
[[0.0, 0.0],
 [0.0464, 0.0464],
 [0.0929, 0.0929],
 [0.1393, 0.1393],
 [0.1858, 0.1858],
 [0.2322, 0.2322],
 [0.2786, 0.2786],
 [0.3251, 0.3251],
 [0.3715, 0.3715],
 [0.418, 0.418]]
```

This seems to come from an error at [this line](https://github.com/marl/medleydb/blob/master/medleydb/multitrack.py#L361), or possibly in the parsing of the confidence file.

**Comment by rabitt** [comment]
That's because stem 0 doesn't exist. This should raise an exception though... Will look into it.

**Comment by rabitt** [comment]
Actually, this is fixed in v1.2
